src/_core/components/MouseFollower/MouseCoordinates.js

```diff
@@ -8,29 +8,16 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
-import Typography from "material-ui/Typography";
-import MiscUtil from "_core/utils/MiscUtil";
-import MapUtil from "_core/utils/MapUtil";
-import styles from "_core/components/MouseFollower/MouseCoordinates.scss";
+import { LonLatCoordinates } from "_core/components/Reusables";
 
 export class MouseCoordinates extends Component {
```

Comment:
So now this is more of a LatLonCoordinatesContainer?

Reply:
yeah, but that name is quite ungainly